samples/net/cellular_modem/prj.conf

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2023 Bjarki Arge Andreasen
+# SPDX-License-Identifier: Apache-2.0
+
+# Networking
+CONFIG_NETWORKING=y
+CONFIG_NET_NATIVE=y
+CONFIG_NET_L2_PPP=y
+CONFIG_NET_IPV4=y
+CONFIG_NET_UDP=y
+CONFIG_NET_SOCKETS=y
+
+# DNS
+CONFIG_DNS_RESOLVER=y
+CONFIG_DNS_SERVER_IP_ADDRESSES=y
+
+# Linodes DNS servers which host test-endpoint.com
+CONFIG_DNS_SERVER1="162.159.27.72"
+CONFIG_DNS_SERVER2="162.159.24.39"
+CONFIG_DNS_SERVER3="162.159.25.129"
+CONFIG_DNS_SERVER4="162.159.26.99"
+CONFIG_DNS_SERVER5="162.159.24.25"
```

Comment:
I don't think this is needed. Let the DNS server provided by the cell network be used.

Reply:
We can switch to using the negotiated DNS server